openedx_events/bridge/avro_attrs_bridge.py

```diff
@@ -23,62 +24,38 @@ class AvroAttrsBridge:
 
     # default extensions, can be overwriteen by passing in extensions during obj initialization
     default_extensions = {
-        DatetimeAvroAttrsBridgeExtension.cls: DatetimeAvroAttrsBridgeExtension()
-    }
-    # default config, should be overwritten by passing in config during obj initialization
-    default_config = {
-        "source": "/openedx/unknown/avro_attrs_bridge",
-        "sourcehost": "unknown",
-        "type": "org.openedx.test.test.test.v0",
+        DatetimeAvroAttrsBridgeExtension.cls: DatetimeAvroAttrsBridgeExtension(),
+        CourseKeyAvroAttrsBridgeExtension.cls: CourseKeyAvroAttrsBridgeExtension(),
     }
 
-    def __init__(self, attrs_cls, extensions=None, config=None):
+    def __init__(self, signal_cls, extensions=None):
```

Comment:
Rename this to what the definition of OpenEdxPublicSignal is for specific events.